patches/native-base+2.13.13.patch

```diff
@@ -37,3 +37,68 @@ index 6620989..3974f66 100644
 \ No newline at end of file
+diff --git a/node_modules/native-base/dist/src/basic/Header.js b/node_modules/native-base/dist/src/basic/Header.js
```

Comment:
Could report the details and the motivation behind this patch in the following file?
https://github.com/pagopa/io-app/blob/master/patches/patches.md

Reply:
To make sure that the entire package was consistent I patched the javascript sources and built the library again with `npm compile` (a script included in the `package.json`), so I included also the minified diff in the patch. I wasn't sure if it was needed (because the already existing patch refers to files in the `dist` subfolder), so I played safe.